### DISCUSSION TITLE
Get project by ID globally

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#7599] [Q&A] [answered]
Hi there!

As I found in the API documentation, [getting a project by ID](https://zitadel.com/docs/apis/resources/mgmt/management-service-get-project-by-id) is possible only if I know the org ID.

Can I get a project only if I know only its ID?

Thank you!

**Comment by hifabienne** [comment]
Hi @ashep 
The management API is based on organization, so you always need the context of the organization you like to access.
If you do not send an organization, per default we take the organization where the user belongs to.

**Reply by ashep** [reply]
As I understand, project IDs are unique across the instance, so it looks logical not to require an org ID to find a project by ID. Moreover, as I see in V2 API, it is possible to [get users by ID](https://zitadel.com/docs/apis/resources/user_service/user-service-get-user-by-id) globally without specifying an org ID, so I suppose something similar could be possible with projects as well.

**Reply by ashep** [reply]
> I suppose something similar could be possible with projects as well.

So the question is, are there any plans to provide such API?

Thank you.

**Reply by hifabienne** [reply, marked as answer]
Yes but this will take some time.
So at the moment we do have our V1 APIs which are service based (Admin, Auth, Mgmt)
And we transition to a resource based API (V2) model, Users, Sessions, Projects, Orgs, etc.
We have started with the first resouorces Users and Session and will start moving other resources soon.
You can track the state here: https://github.com/zitadel/zitadel/issues/6305

**Reply by ashep** [reply]
Thank you!
